modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/configuration/DistributionZonesConfigurationSchema.java

```diff
@@ -29,9 +35,15 @@
 public class DistributionZonesConfigurationSchema {
     /** Global integer id counter. Used as an auto-increment counter to generate integer identifiers for distribution zone. */
     @Value(hasDefault = true)
-    public int globalIdCounter = 0;
+    public int globalIdCounter = DEFAULT_ZONE_ID;
+
+    /** Default distribution zone. */
+    @Name(DEFAULT_ZONE_NAME)
+    @ConfigValue
+    public DistributionZoneConfigurationSchema defaultDistributionZone;
 
-    /** List of configured distribution zones. */
+    /** Other distribution zones. */
+    @ExceptKeys(DEFAULT_ZONE_NAME)
```

Comment:
Could you please explain what @ExceptKeys does?

Reply:
Discussed in PM.